scripts/ci_build.sh

```diff
@@ -28,18 +28,18 @@ if [ -z "$HOME" ]; then
     mkdir -p $HOME
 fi
 
+# Set Docker Container name to be unique
+container_name=""
 
+#Jenkins
+[ ! -z "$BUILD_TAG" ] && container_name="$BUILD_TAG"
+#VSO
+[ ! -z "$BUILD_BUILDID" ] && container_name="$BUILD_BUILDID"
 
-if [[ "$OSNAME" == "ubuntu" ]]; then
-    # Set Docker Container name to be unique
-    container_name=""
+export DOTNET_BUILD_CONTAINER_NAME="$container_name"
```

Comment:
Should we check for container_name being blank [not Jenkins or vso] here? Or is the resultant error message clear enough? I'm worried about strange/hard to debug errors since we're crossing into the docker context here.